### ISSUE TITLE
Vbox7.com broken

### ISSUE THREAD

**Issue by ghost** [ytdl-org/youtube-dl#14685]
- [ x] I've **verified** and **I assure** that I'm running youtube-dl **2017.10.29**
- [x ] At least skimmed through the [README]
- [x ] [Searched] the bugtracker for similar issues including closed ones

### What is the purpose of your *issue*?
- [x ] Bug report (encountered problems with youtube-dl)
```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', '-F', 'https://www.vbox7.com/play:a5e5215ee0']
[debug] Encodings: locale cp1251, fs mbcs, out cp866, pref cp1251
[debug] youtube-dl version 2017.10.29
[debug] Python version 3.4.4 - Windows-10-10.0.16299
[debug] exe versions: ffmpeg N-88555-g6ea7711532, ffprobe N-88555-g6ea7711532, rtmpdump 2.4
[debug] Proxy map: {}
[debug] Using fake IP 95.42.193.161 (BG) as X-Forwarded-For.
[Vbox7] a5e5215ee0: Downloading JSON metadata
[Vbox7] a5e5215ee0: Downloading webpage
[info] Available formats for a5e5215ee0:
format code  extension  resolution note
0            mpd        unknown
```
---
### Description of your *issue*, suggested solution and other information

Seems like Vbox7.com switched to mpd playlist, instead of direct files. Youtube-dl is able to grab the mpd, but this file is useless as it's playlist. Needs rework to get the filenames from mpd (MPD file Generated with GPAC version 0.7.1-revrelease)

New files are split - set1_init, dash_track1 (video) & dash_track2 (audio). They will need proper merge.

**Comment by ubone** [comment]
still broken

**Comment by dirkf** [comment]
With PR #32710:
```console
$ python -m youtube_dl -vF 'https://www.vbox7.com/play:a5e5215ee0'
[debug] System config: [u'--prefer-ffmpeg']
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: [u'-vF', u'https://www.vbox7.com/play:a5e5215ee0']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2021.12.17
[debug] Git HEAD: 630da9eb7
[debug] Python 2.7.18 (CPython i686 32bit) - Linux-4.4.0-210-generic-i686-with-Ubuntu-16.04-xenial - OpenSSL 1.1.1w  11 Sep 2023 - glibc 2.15
[debug] exe versions: avconv 4.3, avprobe 4.3, ffmpeg 4.3, ffprobe 4.3
[debug] Proxy map: {}
[Vbox7] a5e5215ee0: Downloading JSON metadata
[Vbox7] a5e5215ee0: Downloading MPD manifest
[Vbox7] a5e5215ee0: Downloading webpage
[info] Available formats for a5e5215ee0:
format code  extension  resolution note
dash-5       m4a        audio only [eng] DASH audio  134k , m4a_dash container, mp4a.40.2
dash-4       mp4        256x144    [eng] DASH video  132k , mp4_dash container, avc3.64000C, video only
dash-3       mp4        426x240    [eng] DASH video  390k , mp4_dash container, avc3.640015, video only
dash-2       mp4        852x480    [eng] DASH video  733k , mp4_dash container, avc3.64001F, video only
dash-1       mp4        1280x720   [eng] DASH video 1719k , mp4_dash container, avc3.64001F, video only
144p         mp4        144p       
240p         mp4        240p       
480p         mp4        480p       
720p         mp4        720p       (best)
$
```
